corehq/apps/zapier/api/v0_5.py

```diff
@@ -0,0 +1,20 @@
+from corehq.apps.api.resources.v0_4 import XFormInstanceResource
+from corehq.apps.export.system_properties import MAIN_FORM_TABLE_PROPERTIES
+
+
+class ZapierXFormInstanceResource(XFormInstanceResource):
+
+    def dehydrate(self, bundle):
+        for form_property in MAIN_FORM_TABLE_PROPERTIES:
```

Comment:
i think it could be good to add a comment explaining this, at a minimum, and possibly better to extract into a function and then also maybe write a quick test for it.